nrcatalogtools/lvc.py

```diff
@@ -224,3 +224,462 @@ def get_strain_from_lvcnr_file(path_to_file,
         spin_args=spin_args,
         fixed_args=fixed_args,
     )
+
+
+
+
+##################################
+# Generate time domain waveforms
+##################################
+
+def Norm(vec_x, vec_y, vec_z):
+    ''' Compute the norm of a vector.
+
+    Parameters
+    ----------
+    vec_x, vec_y, vec_z : float
+                          The components of a three-vector
+
+    Returns
+    -------
+    norm : float
+           The norm of the three-vector.
+    '''
+
+    vec = np.array([vec_x, vec_y, vec_z])
+
+    return np.sqrt(np.dot(vec, vec))
+
+def Normalize(vec_x, vec_y, vec_z):
+    ''' Normalize a vector.
+
+    Parameters
+    ----------
+    vec_x, vec_y, vec_z : float
+                          The components of a three-vector
+
+    Returns
+    -------
+    vec_x, vec_y, vec_z : float
+                          The normalized components of the three-vector.
+    '''
+
+    norm_vec = Norm(vec_x, vec_y, vec_z)
+
+    return vec_x/norm_vec, vec_y/norm_vec, vec_z/norm_vec\
+
+def CrossProduct(vec1_x, vec1_y, vec1_z, vec2_x, vec2_y, vec2_z):
```

Comment:
Can be replaced by https://numpy.org/doc/stable/reference/generated/numpy.cross.html

ie `CrossProduct(vec1_x, vec1_y, vec1_z, vec2_x, vec2_y, vec2_z)` is the same as `numpy.cross([vec1_x, vec1_y, vec1_z], [vec2_x, vec2_y, vec2_z])`